contrib/gce/configure.sh

```diff
@@ -53,9 +71,14 @@ fetch_env() {
     fi
     echo "${tmp_env_content}" > "${tmp_env_file}"
     # Convert the yaml format file into a shell-style file.
-    eval $(python -c '''
+    eval $(${PYTHON} -c '''
```

Comment:
Looks like this is the same change we had in:
https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure.sh#L67-L75

So this is good.

Reply:
I added the file in the conversion too so people know where it is from.